site/static/examples/tas/sample-job-preferred.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   labels:
     kueue.x-k8s.io/queue-name: tas-user-queue
 spec:
-  parallelism: 40
-  completions: 40
+  parallelism: 4
```

Comment:
why this example is modified? Can we update it in a follow PR for documentation update?

Reply:
Removed